OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -205,7 +205,7 @@ void ConnectionStateChanged(OrderManager om, string password, NetworkConnection
 					{
 						{ "initialMap", lastUpdatedUid ?? map.Uid },
 						{ "initialTab", MapClassification.System },
-						{ "onExit", Game.IsHost ? new Action(() => UpdateSelectedMap()) : null },
+						{ "onExit", Game.IsHost ? (Action)UpdateSelectedMap : modData.MapCache.UpdateMaps },
```

Comment:
Unless I'm missing something, `UpdateSelectedMap` doesn't seem to update the map cache. So why is it necessary to do this for non-hosts?

Reply:
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Map/MapCache.cs#L370-L383

Reply:
Oh I see.